pkgs/by-name/gd/gdlauncher-carbon/package.nix

```
@@ -0,0 +1,132 @@
+{
+  lib,
+  stdenv,
+  appimageTools,
+  fetchurl,
+  graphicsmagick,
+  makeWrapper,
+  copyDesktopItems,
+  autoPatchelfHook,
+  xorg,
+  libpulseaudio,
+  libGL,
+  udev,
+  xdg-utils,
+  electron,
+  addDriverRunpath,
+  makeDesktopItem,
+
+  jdk8,
+  jdk17,
+  jdk21,
+  jdks ? [
+    jdk8
+    jdk17
+    jdk21
+  ],
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gdlauncher-carbon";
+  version = "2.0.20";
+
+  src = appimageTools.extract {
+    inherit (finalAttrs) pname version;
+    src = fetchurl {
+      url = "https://cdn-raw.gdl.gg/launcher/GDLauncher__${finalAttrs.version}__linux__x64.AppImage";
+      hash = "sha256-tI9RU8qO3MHbImOGw2Wl1dksNbhqrYFyGemqms8aAio=";
+    };
+  };
+
+  nativeBuildInputs = [
+    graphicsmagick
+    makeWrapper
+    copyDesktopItems
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    xorg.libxcb
+  ];
```

```suggestion
  buildInputs = [
    xorg.libxcb
    stdenv.cc.cc.lib
  ];

  strictDeps = true;
```
One of the binaries requires libgcc_s. Enable `strictDeps` to make sure we are explicit about dependencies.